test/over_react/component/_deprecated/shared_error_boundary_tests.dart

```diff
@@ -950,16 +950,16 @@ void sharedErrorBoundaryTests(BuilderOnlyUiFactory<ErrorBoundaryPropsMixin> buil
   });
 
   group('logs errors using a `logger`', () {
-    List<Map<String, List>> calls;
-    List<LogRecord> logRecords;
+    late List<Map<String, List>> calls;
+    List<LogRecord>? logRecords;
```

Comment:
Because this is initialized in multiple different tests using the `sharedSetup` utility instead of in a `setUp`, it feels safer to leave this one as nullable, so that in can be nulled out and prevent test pollution if a test forgets to initialize it.